app/controllers/post_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PostController < ApplicationController
```

Comment:
- [ ] The controller, just like the routes should use plural form, instead of singular.

  ```
  class PostsController < ApplicationController
  ```

- [ ] Kindly rename the file also from `post_controller.rb` to `posts_controller.rb`.